### PR TITLE
chore(build): upgrade to TS 1.6

### PR DESCRIPTION
**Pull request by alexeagle**
Needed before angular can upgrade to TS 1.6

### CODE REVIEW

File: lib/main.ts [hunk 1]
```diff
@@ -48,6 +47,7 @@ export const COMPILER_OPTIONS: ts.CompilerOptions = {
   experimentalDecorators: true,
   module: ts.ModuleKind.CommonJS,
   target: ts.ScriptTarget.ES5,
+  moduleResolution: ts.ModuleResolutionKind.Classic,
```
Comment (IgorMinar) [inline lib/main.ts]: do you need this for ts2dart? I don't think that you have any named imports pointing to your source directory

File: tsconfig.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "compilerOptions": {
+  "compilerOptions": {
```
Comment (IgorMinar) [inline tsconfig.json]: trailing ws

File: lib/main.ts (hunk 1 above)
Comment (IgorMinar) [inline lib/main.ts]: resolved in person.. still needed

**Comment by IgorMinar** [comment]
lgtm

**Comment by alexeagle** [comment]
merged in 01420c0

**Pull request closed without merging**